### DISCUSSION TITLE
get imu data

### DISCUSSION THREAD

**Discussion by powehi2020** [bitcraze/discussions#393] [Q&A]
hi～ I have read the readme.md which write 'imu_tests.MPU6050'  but it does show an example and in the code of crazyflie-lib-python,also can search the message of imu. How can i get the imu data ? can anyone give me some advice? thank you very much!

**Comment by knmcguire** [comment]
Hi!

Easiest to read out IMU data, is to go to [the plotter tab of the cfclient ](https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/userguides/userguide_client/plotter_tab/) by going to 'view' > 'tabs' > check 'Plotter' and check out the 'stabilizer' config for roll pitch yaw. 

If you are looking for other data of the IMU, check out which [logs ](https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/api/logs/)you are looking for. Then configure that for a [log block in the cfclient](https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/userguides/userguide_client/#logging) and then plot the result in the plotter tab
